nextflow/accession.nf

```diff
@@ -217,8 +217,8 @@ process csi_index_vcf {
 
     """
     cd $params.public_dir
-    # remove the uncompressed vcf file
-    rm ${accessioned_vcfs.join(' ')}
+    # remove the uncompressed accessioned vcf file, if present
+    ${accessioned_vcfs.size() > 0 ? "rm ${accessioned_vcfs.join(' ')}" : ""}
```

Comment:
I admire the dexterity but for the record I would prefer if we sticked to one language per section: inside the quote it's bash and outside it's groovy.

Reply:
Yeah that's fair enough, technically the `accessioned_vcfs.join(' ')` was already groovy but this is definitely more confusing.  I'll just use a conditional like we agreed to do in Nextflow...